admin/onpage/class-onpage-endpoint.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @package WPSEO\Admin\OnPage
+ */
+
+/**
+ * Class WPSEO_OnPage_Endpoint
+ */
+class WPSEO_OnPage_Endpoint {
```

Comment:
I would rename this to `WPSEO_Ryte_Endpoint` because this is new code and `OnPage.org` has been renamed to `Ryte`

Reply:
The rest of the codebase still uses the `OnPage` both in classnames as in database options.

Having a `WPSEO_Ryte_Option` class which saves to the `wpseo_onpage` option_name would be confusing.

I think it's better to keep the endpoint and service names consistent with what they're serving.